public/blog/first-ever-truffle-retreat.md

```diff
@@ -0,0 +1,123 @@
+
+![Truffle employees in September](/blog/images/first-ever-truffle-retreat/truffle-map-01.jpg)
+
+When we brought Nick and Josh onto the team, it became obvious that we needed to meet up in person to solidify our roles, define responsibilities, and establish our own culture before growing the team even larger. 
+
+And we wanted to hang out in the woods, at the top of a mountain, and get to know each other. 
+
+## The Location
+
+![View from the house](/blog/images/first-ever-truffle-retreat/location.jpg)
+
+We had our retreat near Victoria, BC where John lives. Nick, Josh and I traveled all day through a series of planes, ferries and car rides to finally arrive at the house around 7 pm. The house sits atop a large hill in Braemer Heights overlooking all of Victoria, and on a good day has a beautiful view of Mt. Baker in the States. 
+
+The house was the definition of excess for a team of four, especially with only three of us sleeping there. It had five bedrooms and four bathrooms; a deck with a grill; three stories, the topmost containing only an office with 360-degree windows; and loads of wildlife. Unexpectedly, it was only ten minutes away from a Costco, which helped immensely. Even more unexpectedly, this mansion cost us less than three hotel rooms and a coworking space.  
+
+![Another view](/blog/images/first-ever-truffle-retreat/mountain.jpg)
+
+## The Work
+
+I had a few specific goals for our retreat: to define everyone’s roles and responsibilities within our team, to define our product roadmap for the next six months, and to create a mission statement for Truffle that our whole team could rally behind. 
+
+### Retrospective
+
+We started with a retro. We wanted to evaluate where we were as a team before deciding where we wanted to go in the future.
+
+So on Tuesday morning, we sat down and gave praise for what we’ve done well and identified areas where we could improve. I brought a list of business areas that are important for Truffle -- things like development, technical leadership, branding, marketing, etc. -- and we graded ourselves on each of them. 
+
+This turned out to be an interesting exercise. As the leader of the team you want to say “we’re doing great”, and not diminish anyone’s efforts -- because, it’s true, we’ve come far for a small team. However, we need to know where we can improve and how different areas of our company, teamwork and products stack up to the rest. In general, we all seemed to agree on the results: Branding and growth have been stellar (e.g., we have a good product that people like to use), but more external communication -- like our technical leadership, our relationship with the Ethereum Foundation, and relationships with some of our VIP users -- could use some improvement. Though these results were mostly subjective judgements, the conversations that resulted helped inform discussions about our roles and future roadmap. 
```

Comment:
Small points:
* If we're being consistent, it's "company, teamwork, and products"
* I try to avoid "e.g." as not everyone knows what it means. Suggest "as in".
* We have VIP users?